### DISCUSSION TITLE
Pruned project always the same hash

### DISCUSSION THREAD

**Discussion by lillheaton** [vercel/turborepo#4136] [Help] [answered]
Is it possible to get a unique hash on a pruned project?

Steps to reproduce

1. Prune project (`npx turbo prune --scope=myapp`
2. cd ./out
3. Install dependencies (I use npm, so npm ci)
4. turbo run build --filter=myapp --dry=json

The hash is always the same, even if I edit and modify the files within myapp.

If I run turbo run build --filter=myapp --dry=json on my root folder (so not within the pruned folder ./out) I get new hashes as expected.

**Comment by anthonyshew** [comment, marked as answer]
This sounds like a mis-use of `turbo`. The `prune` command is meant for CI builds rather than local development. In that situation, there wouldn't be the possibility for editing source code files (or, at least, there *shouldn't* be).

If you have a use case for this that we haven't thought of, could you open an issue? Thank you!

(For context on why those hashes don't change, I believe it's because we find the root of your repo at your project's root rather than the `out` "root". So `turbo` is working as expected since the code in `out` isn't a part of your workspace - it has it's own "root". However, as mentioned, the happy path for `turbo` would have you not doing this in the first place). 😄
